ci-operator/jobs/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main-periodics.yaml

```diff
@@ -9,17 +9,19 @@ periodics:
     repo: ocs-operator
   labels:
     ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: latest-upstream
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-red-hat-storage-ocs-operator-main-ocs-operator-bundle-master-e2e-aws
+  name: periodic-ci-red-hat-storage-ocs-operator-main-latest-upstream-ocs-operator-bundle-e2e-aws
```

Comment:
Note how `latest-upstream-` is added here because of the `variant` information, not because the test sets the name as such. (ref: https://github.com/openshift/release/pull/21168/files#r706488659)

Reply:
:+1: